packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -926,11 +926,11 @@ abstract class ResidentHandlers {
     final List<FlutterView> views = await flutterDevices
       .first!
       .vmService!.getFlutterViews();
-    final Map<String, Object> data = await (flutterDevices.first!.vmService!.getSkSLs(
+    final Map<String, Object?>? data = await flutterDevices.first!.vmService!.getSkSLs(
```

Comment:
`await flutterDevices.first!.vmService!` is called on 2 lines in a row, make a local.

Reply:
:+1: